src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/NotifyService.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
+
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import uk.gov.service.notify.NotificationClient
+
+@Service
+class NotifyService(
+  @Value("\${notify.enabled}") private val enabled: Boolean,
+  @Value("\${notify.templates.licence-approved-id}") private val licenceApprovedTemplateId: String,
```

Comment:
Not sure that passing these `@Value`s as a parameter is the right thing to. I think usually they would just be class properties. If the reason for passing them in as a parameter is to help with unit tests, you can use ReflectionTestUtils which mock the `application.yml` values